app/models/tabular_import.rb

```diff
@@ -35,10 +35,8 @@ def failed?
   protected
 
   def open_sheet
-    self.sheet = Roo::Spreadsheet.open(file).sheet(0)
-  rescue TypeError, ArgumentError => e
-    raise e unless /not an Excel 2007 file|Can't detect the type/.match?(e.to_s)
-    add_run_error(:wrong_type)
+    self.file = Paperclip.io_adapters.for(file).read if file.class == Paperclip::Attachment
```

Comment:
I don't love this but it will be going away soon. This is required for feature specs only.